ee/tabby-db/src/user_completions.rs

```diff
@@ -149,11 +149,13 @@ impl DbConn {
             SELECT CAST(STRFTIME('%s', DATE(created_at)) AS TIMESTAMP) as start,
                    SUM(1) as completions,
                    SUM(selects) as selects
-            FROM user_completions
-            WHERE created_at >= ?1 AND created_at < ?2
-                AND ({no_selected_users} OR user_id IN ({users}))
-                AND (({no_selected_languages} OR language IN ({languages}))
-                     OR (language NOT IN ({not_languages})))
+            FROM (
+                SELECT created_at, selects, IIF(language IN ({all_languages}), language, 'other') as language
```

Comment:
```suggestion
                SELECT created_at, selects, IF(language IN ({all_languages}), language, 'other') as language
```

Reply:
`IF` does not exist in sqlite, the function is called `IIF`